packages/medusa-payment-adyen/src/services/adyen.js

```diff
@@ -150,12 +135,12 @@ class AdyenService extends BaseService {
 
   /**
    * Status for Adyen payment.
-   * @param {object} paymentData - payment method data from cart
-   * @returns {string} the status of the payment
+   * @param {Record<string, unknown>} paymentData - payment method data from cart
+   * @returns {Promise<PaymentSessionStatus>} the status of the payment
    */
-  getStatus(paymentData) {
+  async getStatus(paymentData) {
```

Comment:
question: This method does not need to be an async one as far as I can see. What's your reasoning behind adding it?

Reply:
Basically, the getSatus from the payment-provider was already async, so I thought that giving the ability to the consumer to have some third party call on async processing of the status could be something interesting. But I might be wrong, let me know wdyt

Reply:
Got it! Also fine with keeping it async 👍